BankUsers/Client.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BankUsers
+{
+    public class Client : User
+    {
+        public bool IsVipOneClient { get; set; }
+        
+
+        public Client(string firstName, string lastName, int age, bool isVipClient=false) : base(firstName, lastName, age)
+        {
+            IsVipOneClient = false;
```

Comment:
You need to use parameter isVipClient

```suggestion
            IsVipOneClient = isVipClient;
```